local-cli/bundle/bundleCommandLineArgs.js

```diff
@@ -57,5 +57,9 @@ module.exports = [
     command: 'verbose',
     description: 'Enables logging',
     default: false,
+  }, {
+    command: 'reset-cache',
+    description: 'Removes cached files',
+    default: false
```

Comment:
yeah. `react-native-xcode.sh` sounds like a reasonable place. we should add it to `android` counterpart also then.